InteractiveHtmlBom/web/ibom.js

```diff
@@ -836,6 +853,37 @@ function prepCheckboxes() {
   }
 }
 
+function populateDarkenWhenCheckedOptions() {
+  var container = document.getElementById("darkenWhenCheckedContainer");
+  container.innerHTML = '';
+  
+  function createOption(name, displayName) {
+    var id = "darkenWhenChecked-" + name;
+
+    var div = document.createElement("div");
+    div.classList.add("radio-container");
+
+    var input = document.createElement("input");
+    input.type = "radio";
+    input.name = "darkenWhenChecked";
+    input.value = name;
+    input.id = id;
+    input.onchange = setDarkenWhenChecked.bind(undefined, name);
```

Comment:
Is this equivalent of () => setDarkenWhenChecked(name); ? 
If so I prefer lambda.

Reply:
Yes, done.